File: C++/427.cpp
#include <iostream>

using namespace std;

// 427 - Yo Soy Tu... - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for(int i = 0; i<loop; i++) {
        string luke = "";
        string father = "";
        cin >> luke;
        cin >> father;
        if (luke == "Luke" && father == "padre") {
            cout << "TOP SECRET" << endl;
        } else {
        cout << luke + ", yo soy tu " + father << endl;
        }
    }

}

File: Java/427.java
import java.util.Scanner;

/**
 *
 * 427 - Yo Soy Tu... - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int casos = sc.nextInt();
        for (int i = 1; i <= casos; i++) {
            String nombre = sc.next();
            String parentesco = sc.next();
            if (nombre.equals("Luke") && parentesco.equals("padre")) {
                System.out.println("TOP SECRET");
            } else {
                System.out.println(nombre + ", yo soy tu " + parentesco);
            }
        }
    }
}
